librz/search/regexp.c

```diff
@@ -32,12 +32,12 @@ RZ_API int rz_search_regexp_update(RzSearch *s, ut64 from, const ut8 *buf, int l
 			return -1;
 		}
 
-		matches = rz_regex_match_all_not_grouped(compiled, (char *)buf, len, from, RZ_REGEX_DEFAULT);
+		matches = rz_regex_match_all_not_grouped(compiled, (char *)buf, len, 0, RZ_REGEX_DEFAULT);
```

Comment:
```suggestion
		matches = rz_regex_match_all_not_grouped(compiled, (const char *)buf, len, 0, RZ_REGEX_DEFAULT);
```

Is this possible btw?

Reply:
I believe so